source/includes/concept/_widget.md

```diff
@@ -0,0 +1,17 @@
+## Widget
+
+You have fetched your data from your [dataset](#dataset) using [queries](#query). And you know how to build custom visualizations for geospatial data using [layers](#layer). Yet, sometimes your data is not geo-located, and you want to build a plain graphical representation of it: this is when **widgets** come in handy.
+
+A **widget** is a visual specification of how to style and render the data of a dataset (think pie, bar or line charts).
+
+As with layers, each widget has a single dataset associated with it, and a dataset can be associated with many widgets. You can represent the same data in different ways by creating different widgets for the same dataset. The same widget can store independent configuration values for each RW API based application. It can also contain the required configuration for rendering the same visualization using different rendering tools.
+
+Like layers, the widget itself does not interact with the dataset data. You can either use the widget's `queryUrl` field to store the query to get the widget's data or store it inside the free form `widgetConfig` object. In any of these cases, it is your responsibility as an API user to query the data that will be used for rendering the widget.
+
+In the [widget endpoint documentation](#widget7), you can get into more detail on how you can manage widgets.
+
+### Widget configuration using Vega grammar
+
+As in the case of layers (where many fields are free-form), the RW API does not apply any restriction to whatever is saved in the widget configuration field (`widgetConfig`). This allows for a very high level of flexibility for users, but has the downside of making it harder to document how to use widgets.
+
+However, contrary to what happens with layers, many of the existing widgets base themselves in the [Vega grammar](https://github.com/vega/vega) for configuring its graphical representations. Vega is a visualization grammar, a declarative format for creating interactive visualization designs. You can then use a chart library that supports the Vega grammar (such as [d3.js](https://d3js.org/)) to render the widget and obtain your graphical representation.
```

Comment:
I would take a stronger stance here and say that Vega is the standard convention, but that you can use anything you want.
Also, really like that you describe the full use case here, even if it's just a couple of sentences. Exactly what concept docs are for!

Reply:
Reworded things a bit to say that the existing widgets' standard approach for building widget configuration objects is using Vega. Also, added a note at the end saying that users can still use whatever configuration format they wish :+1: